client-ts/Microsoft.AspNetCore.SignalR.Client.TS/IConnection.ts

```diff
@@ -0,0 +1,10 @@
+import { DataReceived, ConnectionClosed } from "./Common"
+
+export interface IConnection {
+    start(transportName: string): Promise<void>;
```

Comment:
I am planning to resurrect a community PR that replaced string with enum and possibly introduce a factory if this helps with testing.